File: src/org/apache/commons/logging/Log.java

package org.apache.commons.logging;

public interface Log {
    void debug(Object o);

    void debug(Object o, Throwable throwable);

    void error(Object o);

    void error(Object o, Throwable throwable);

    void fatal(Object o);

    void fatal(Object o, Throwable throwable);

    void info(Object o);

    void info(Object o, Throwable throwable);

    boolean isDebugEnabled();

    boolean isErrorEnabled();

    boolean isFatalEnabled();

    boolean isInfoEnabled();

    boolean isTraceEnabled();

    boolean isWarnEnabled();

    void trace(Object o);

    void trace(Object o, Throwable throwable);

    void warn(Object o);

    void warn(Object o, Throwable throwable);
}

File: src/org/apache/commons/logging/Log.kt
package org.apache.commons.logging

interface Log {
    fun debug(o: Any?)
    fun debug(o: Any?, throwable: Throwable?)
    fun error(o: Any?)
    fun error(o: Any?, throwable: Throwable?)
    fun fatal(o: Any?)
    fun fatal(o: Any?, throwable: Throwable?)
    fun info(msg: Any?)
    fun info(o: Any?, throwable: Throwable?)
    fun isDebugEnabled(): Boolean
    fun isErrorEnabled(): Boolean
    fun isFatalEnabled(): Boolean
    fun isInfoEnabled(): Boolean
    fun isTraceEnabled(): Boolean
    fun isWarnEnabled(): Boolean
    fun trace(o: Any?)
    fun trace(o: Any?, throwable: Throwable?)
    fun warn(o: Any?)
    fun warn(o: Any?, throwable: Throwable?)
}
